### PR TITLE
Add USD value to collection Activity table

### PR DESCRIPTION
**Pull request by mmnavarr**
## This PR Includes the following changes

- Remove the text `$MAGIC` from the Activity table header since the field contains both MAGIC/USD values
- Added calculation of current USD value of the activity txn price and display it below the MAGIC quantity (see screenshot below)
- Added a new `formatNumberUsd` util function to facilitate USD display

image

cc: @wyze @jcheese1

### CODE REVIEW

File: src/components/Listings.tsx [hunk 1]
```diff
@@ -186,7 +189,19 @@ const Listings = ({
...
-                        {formatPrice(listing.pricePerItem)}
+                        <div className="flex items-center">
+                          {formatPrice(listing.pricePerItem)}
+                          <div className="text-xs px-1 text-gray-800 dark:text-gray-700">
+                            MAGIC
+                          </div>
+                        </div>
+                        <div className="text-gray-500 text-xs">
+                          $
+                          {formatPriceUsd(
```
Comment (jcheese1) [inline src/components/Listings.tsx:200]: would `formatPrice` do the same thing? It's also in utils.ts

Comment (mmnavarr) [reply src/components/Listings.tsx]: The issue with `formatPrice` is that it does `formatEther` within itself. In this case, I have to do a multiplication computation (MAGIC price of item * USD value of MAGIC) prior which requires me to convert price of magic & its USD equivalent to numbers.

To not overflow I have to do the `formatEther` myself and then the number I have no longer fits into the usage of `formatPrice`. I tried using `formatNumber` but it does not ensure 2 decimal places and so I went with a new helper that prefixes with a `$` and ensures that 2 decimal places are always present.

**Comment by jcheese1** [comment]
@all-contributors please add @mmnavarr for code

**Comment by jcheese1** [comment]
CleanShot 2022-01-06 at 12 56 10@2x
@mmnavarr wna add it for mobile view too?

**Comment by alecananian** [comment]
Closing stale PR

**Pull request closed without merging**